content/posts/building-a-successful-idp-the-human-factor.md

```diff
@@ -0,0 +1,43 @@
++++
+title = 'Building a Successful IDP - The Human Factor'
+date = 2024-03-26T20:14:31+01:00
+draft = true
++++
+
+In the world of software engineering, an Internal Developer Platform (IDP) serves as the backbone of your technological infrastructure, empowering your developers to efficiently build, deploy, and manage applications. However, it's not just the technical stack that determines the success of an IDP; factors like *onboarding*, *ease of use*, and *support* are equally—if not more—important. In smaller organizations, paying attention to these can be the difference between a smooth-running machine and a cluttered, under-utilized platform. Let’s dive into these critical aspects for building a successful IDP.
+
+## Onboarding: Setting Developers Up for Success
+
+Imagine this: A new developer joins your team, ready to contribute, but they hit a documentation wall that's about learning the platform—not executing tasks. Remember, the goal isn't to master the IDP itself, but to deliver products faster. That's why your documentation should be task-driven. Provide step-by-step templates within your CI/CD pipelines that show—not just tell—developers how to deploy, connect to databases, and integrate other services. The documentation should not be a textbook but a playbook, one that developers can run with immediately.
```

Comment:
I am wondering if we can replace platform-not and show-not for somethin else, it's oke but still. 

should not be a textbook but a playbook -> this seems to imply that it should only be a playbook but that is not what we are trying to say here.